src/pt/readmangas/src/eu/kanade/tachiyomi/extension/pt/readmangas/ReadMangas.kt

```diff
@@ -247,13 +247,16 @@ class ReadMangas() : HttpSource() {
 
     override fun pageListParse(response: Response): List<Page> {
         val document = response.asJsoup()
-        val script = document.select("script").map { it.data() }
-            .firstOrNull { IMAGE_URL_REGEX.containsMatchIn(it) }
-            ?: return emptyList()
+        val scripts = document.select("script").map { it.data() }
```

Comment:
This will allow you to run the regex once:

```suggestion
        val scripts = document.select("script").joinToString("\n") { it.data() }
```

Reply:
Fix it